packages/jest-haste-map/src/index.js

```diff
@@ -325,6 +325,28 @@ class HasteMap extends EventEmitter {
           throw new Error(message);
         }
         this._console.warn(message);
```

Comment:
Let's do it as a future step, as this changeset make it no worse than the current situation. I definitely think doing "console.warn/log" in the middle of a library is an anti-pattern and that we should expose it as an error condition through `getModule` instead.